sympy/external/tests/test_numba.py

```diff
@@ -0,0 +1,29 @@
+# This testfile tests SymPy's compatibility with Numba
+
+from sympy.external import import_module
+from sympy import sin, exp, symbols, lambdify, Matrix
+
+numba = import_module('numba')
+if not numba:
+    #bin/test will not execute any tests now
+    disabled = True
+
+def test_jit():
+    # Test that numba.jit works with lambdify
+    x = symbols('x')
+    f1 = lambdify(x, sin(x), 'numpy')
+    assert numba.jit(f1)(1) == f1(1)
```

Comment:
Numba jitted functions don't get the exact same numeric result as NumPy in general. You may want to use `assert abs(numba.jit(f1)(1) - f1(1)) < 1e-15` instead.